frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,54 @@
+.input {
+  padding: 20px 24px;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #000;
+  border: 1px solid #9f18d9;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #9f18d9 0.5;
+}
+
+.input:active {
+  color: #000;
+  outline-style: none;
+}
+
+.input:focus {
+  outline-style: none;
+}
+
+.input:disabled {
+  color: #404040;
+  background: #bfbfbf;
```

Comment:
Let's be more specific and use `background-color` instead of the shorthand `background`